adivinador

brunorz97 opened the pull request:
realizacion de mini juego adivinador

brunorz97 committed:
adivinador

realizacion de mini juego adivinador

brunorz97 committed:
se arregla bug

devtsp commented on adivinar.html:
```diff
@@ -0,0 +1,40 @@
+<br>
+<button type="submit" id="generar-numero">Generar numero aleatorio</button>
+<p>Numero de intentos restantes:<strong id="intentos"></strong></p>
+<label>Introduce un numero:</label>
```
Los label deben llevar un atributo for="" con el id del input al que hacen referencia. En este caso sería 
`<label for="numero">Introduce un numero:</label>`

brunorz97 replied:
gracias! modificado :)

devtsp replied:
Fijate tambien que podes intentar generar un unico div para desplegar el resultado, y con un condicional elegir cual va a ser la propiedad "innerText" de tal div. Si fue correcta la adivinanza, será un mensaje A (Programado dentro de javascript), e lo contrario, sera un mensaje B. Así ahorras líneas y elementos de HTML.

brunorz97 committed:
agregar for

se agrega propiedad a label para asociarlo a su correspondiente input

brunorz97 committed:
optimizacion

el resultado se muestra en un solo div y no en dos como estaba anteriormente, esto optimiza la estructura del html
